yt/data_objects/construction_data_containers.py

```diff
@@ -153,7 +153,7 @@ def _get_cut_mask(self, grid):
         for mi, (i, pos) in enumerate(zip(pids, self.positions[points_in_grid])):
             if not points_in_grid[i]:
                 continue
-            ci = ((pos - grid.LeftEdge) / grid.dds).astype("int")
+            ci = ((pos - grid.LeftEdge) / grid.dds).astype("int64")
```

Comment:
This particular one doesn't necessarily need promotion to int64 (although I recognize numpy may do it internally) because it's a relative offset.

Reply:
this doens't change behaviour, I'm just making the size explicit rather than implicit.